package.json

```diff
@@ -4,7 +4,9 @@
   "description": "A utility function which helps to inline fontawesome SVG files.",
   "main": "index.js",
   "scripts": {
-    "test": "ava"
+    "test": "ava",
+    "test-cov": "nyc --reporter=lcovonly yarn test",
+    "eslint": "eslint"
```

Comment:
This script it not needed, `yarn eslint` by default run the package bin. We don't have any linting rules enabled so it's a no-operation anyway.